packages/mesa.rb

```diff
@@ -27,55 +19,40 @@ class Mesa < Package
   depends_on 'libxfixes'
   depends_on 'libxdamage'
   depends_on 'libxcb'
+  depends_on 'libxvmc'
   depends_on 'libxtrans'
   depends_on 'libxkbfile'
   depends_on 'wayland'
   depends_on 'python27'
   depends_on 'bison'
   depends_on 'libxshmfence'
   depends_on 'elfutils'
+  depends_on 'llvm'
   
   def self.build
-    #system "pip install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} Mako"
+    system "pip uninstall -y Mako MarkupSafe || :"
+    system "pip install --install-option=\"--prefix=#{CREW_DEST_PREFIX}\" Mako"
     system "pip install --install-option=\"--prefix=#{CREW_PREFIX}\" Mako"
-    case ARCH
-      when 'x86_64', 'i686'
-        system "./autogen.sh \
-                --prefix=#{CREW_PREFIX} \
-                --libdir=#{CREW_LIB_PREFIX} \
-                --enable-shared-glapi \
-                --with-gallium-drivers=i915,nouveau,r300,r600,radeonsi,pl111,svga,swrast,swr,tegra,vc4,virgl \
-                --with-dri-drivers=i915,i965,nouveau,radeon,r200,swrast \
-                --enable-osmesa \
-                --enable-opengl \
-                --enable-egl \
-                --enable-gles1 \
-                --enable-gles2 \
-                --with-platforms=x11,drm,wayland \
-                --enable-gbm \
-                --enable-xa \
-                --enable-glx\
-                --enable-dri \
-                --enable-llvm"
-      when 'aarch64', 'armv7l'
-        system "./configure \
-                --prefix=#{CREW_PREFIX} \
-                --libdir=#{CREW_LIB_PREFIX} \
-                --enable-shared-glapi \
-                --with-gallium-drivers=nouveau,r300,freedreno,pl111,swrast,tegra,vc4,virgl \
-                --with-dri-drivers=nouveau,radeon,r200,swrast \
-                --enable-osmesa \
-                --enable-opengl \
-                --enable-egl \
-                --enable-gles1 \
-                --enable-gles2 \
-                --with-platforms=x11,drm,wayland \
-                --enable-gbm \
-                --enable-xa \
-                --enable-glx\
-                --enable-dri \
-                --enable-llvm"
-      end
+    system "./configure",
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           "--enable-shared-glapi",
+           "--with-gallium-drivers=i915,nouveau,r300,r600,radeonsi,pl111,svga,swrast,swr,tegra,vc4,virgl,freedreno",
+           "--with-dri-drivers=i915,i965,nouveau,radeon,r200,swrast",
```

Comment:
Why do you always want to change from the original?  We should *NOT* be installing i915 or i965 drivers on arm.  Those are Intel drivers.  If you had an arm system, I might let this slide but you don't so please return to the previous configure options.

Reply:
How about you test it? If it works, then this can be resolved.

Reply:
Just return it back to the way it was please.  I don't have time to reverse things you mess around with.

Reply:
Actually, I remember why I added the `--disable-asm` option. So stuff like this wouldn't happen.

Reply:
All I'm doing is asking you to test this on ARM before it gets merged. Could you do that for me?